app/models/backend_api.rb

```diff
@@ -38,7 +38,7 @@ class BackendApi < ApplicationRecord
 
   alias_attribute :api_backend, :private_endpoint
 
-  before_validation :set_private_endpoint, :set_port_private_endpoint
```

Comment:
Can we get rid of the `private_endpoint` attr then? Not saying that it has to be in this same PR 😄

Reply:
I'll add a TODO just in case